config.yml

```diff
@@ -12,13 +12,13 @@ ruby:
   ## then be sure to bump the corresponding `package_revision`!
   minor_version_packages:
     - minor_version: '3.3'
-      full_version: '3.3.1'
```

Comment:
For minor version packages, please bump the package_revision as well.

Reply:
Ahh, sorry about that. I've pushed the changes